### DISCUSSION TITLE
How I wish there is a prisma endpoints

### DISCUSSION THREAD

**Discussion by sandbox-apps** [prisma/prisma#8380] [Q&A] [answered]
:D

**Comment by ryands17** [comment, marked as answer]
You can follow/comment on these requests as it's similar to your use case:
https://github.com/prisma/prisma/issues/5000
https://github.com/prisma/prisma/issues/100

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
